src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { useState, ReactElement, useEffect } from 'react'
+import {
+  DDO,
+  File as FileMetadata,
+  Logger,
+  ServiceType,
+  publisherTrustedAlgorithm
+} from '@oceanprotocol/lib'
+import { toast } from 'react-toastify'
+import FormStartComputeDataset from './FormComputeDataset'
+import Web3Feedback from '../../../molecules/Wallet/Feedback'
+import Price from '../../../atoms/Price'
+import File from '../../../atoms/File'
+import styles from './index.module.css'
+import Input from '../../../atoms/Input'
+import Alert from '../../../atoms/Alert'
+import { useSiteMetadata } from '../../../../hooks/useSiteMetadata'
+import checkPreviousOrder from '../../../../utils/checkPreviousOrder'
+import { useOcean } from '../../../../providers/Ocean'
+import { useWeb3 } from '../../../../providers/Web3'
+import { usePricing } from '../../../../hooks/usePricing'
+import { useAsset } from '../../../../providers/Asset'
+import {
+  queryMetadata,
+  transformDDOToAssetSelection
+} from '../../../../utils/aquarius'
+import { Formik } from 'formik'
+import {
+  getInitialValues,
+  validationSchema
+} from '../../../../models/FormStartComputeDataset'
+import { AssetSelectionAsset } from '../../../molecules/FormFields/AssetSelection'
+import { SearchQuery } from '@oceanprotocol/lib/dist/node/metadatacache/MetadataCache'
+import axios from 'axios'
+
+export default function Compute({
+  ddo,
+  isBalanceSufficient,
+  dtBalance,
+  file
+}: {
+  ddo: DDO
+  isBalanceSufficient: boolean
+  dtBalance: string
+  file: FileMetadata
+}): ReactElement {
+  const { marketFeeAddress } = useSiteMetadata()
```

Comment:
we have to pass this to the order methods. They both accept a `mpAddress` parameter, hich I think is meant for the `marketFeeAddress` (it's undocumented in ocean.js code)